import-service/lib/import-service-stack.ts

```diff
@@ -85,5 +132,25 @@ export class ImportServiceStack extends cdk.Stack {
     importedBucket.grantRead(importFileParser);
     importedBucket.grantDelete(importFileParser);
     importedBucket.grantPut(importFileParser);
+
+    api.addGatewayResponse("UnauthorizedResponse", {
+      type: apigateway.ResponseType.UNAUTHORIZED,
+      statusCode: "401",
```

Comment:
Instead of adding gateway responses for 401/403 statues, common gateway response type `DEFAULT_4XX` could be used